src/registry_schemas/schemas/business_document.json

```diff
@@ -0,0 +1,181 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/business_document",
+    "type": "object",
+    "title": "The Business Document Schema",
+    "definitions": {
+        "datetimeUTC": {
+            "type": "string",
+            "title": "A datetime string in UTC isoformat.",
+            "format": "date-time",
+            "examples": ["2022-01-11T20:03:33.857807+00:00"]
+        },
+        "amalgamatedEntities": {
+            "type": "array",
+            "items": {
+                "required": ["identifier", "name"],
+                "properties": {
+                    "identifier": { "type": "string" },
```

Comment:
should the `identifier` here follow the same format as `identifier` defined in the business schema?  or is this done intentionally?

Reply:
this is intentional and temporary because the amalgamated entities aren't set correctly yet (for now they are always set as 'Not Available') and same thing with 'name'